script/script.js

```diff
@@ -82,28 +86,80 @@ function removeThis(e){
 }
 
 
-function storeIdea (id, card) {
-	localStorage.setItem(id, JSON.stringify(card));
-}
 
 $('.main-title , .idea-input').on('keyup', checkValue); 
 
-function checkValue(e) {
-	if (e.keyCode === 13 && ($('.main-title').val() !== '' && $('.idea-input').val() !== '')){
+function checkValue(e) {;
+	if (e.keyCode === 13 && $('.main-title').val() !== '' && $('.idea-input').val() !== ''){
+		e.preventDefault();
 		addIdea();
 	}
-};
+}
 
 $('.bookmark-list').on('click', '.upvote-button', upVoteTodo);
 
 function upVoteTodo(){
-	var qualityArr = ['swill', 'plausible', 'genius'];
-	var quality = $(this).parent().find('.quality-content').text();
-    quality > qualityArr[0] ? quality = qualityArr[1] : quality = qualityArr[2]
-    $(this).parent().find('.quality-content').text(quality);
-	console.log(quality)
+	var currentId = ($(this).closest('.idea-article').attr('id'));
+	var currentQuality = ($(this).parent().find('.quality-content').text());
+	var qualityValue = retQualityValue(currentQuality);
+	if (qualityValue === 1){
```

Comment:
This feels a bit odd having an empty if section. There's got to be a a different way. I'll mess with this a bit since we got to also get it done within 8 lines.